workflow/tools/cwl_workflow_runner.py

```diff
@@ -95,17 +95,14 @@
 output_dict = dict()
 
 if repo_name == 'pcawg-gatk-cocleaning':
-    output_file = os.path.join(os.getcwd(), cwl_outdir, 'printreads_tumor/output/*.cleaned.bam')
-    if os.path.isfile(output_file):
-        output_dict['cleaned_tumor_bam'] = output_file[0]
-    else:
-        exit('BQSR CoCleaning failed to produce output for cleaned_tumor_bam!')
-
-    output_file = os.path.join(os.getcwd(), cwl_outdir, 'printreads_normal/output/*.cleaned.bam')
-    if os.path.isfile(output_file):
-        output_dict['cleaned_normal_bam'] = output_file[0]
-    else:
-        exit('BQSR CoCleaning failed to produce output for cleaned_normal_bam!')
+    for bamtype in ['normal', 'tumor']:
+        # get the basename of the input normal/tumor bams
+        bambase = os.path.basename(input_json[bamtype+'_bam']['path']).replace('.bam', '')
+        output_file = os.path.join(os.getcwd(), cwl_outdir, bambase+'.realigned.cleaned.bam')
```

Comment:
as discussed, please check the existence of `.bai` file, and create a link for each `.bai` to link to `.bam.bai`.